### DISCUSSION TITLE
Is there a helix plugin for vscode or jetbrains?

### DISCUSSION THREAD

**Discussion by zhaob1n** [helix-editor/helix#2172] [Q&A]
I'll be excited to see one

**Comment by mauscoelho** [comment]
You can mimic the Helix keybindings using Dance extension on vscode

https://gist.github.com/mauscoelho/5c727b50472f44941b26c8df5a46a0e9
https://marketplace.visualstudio.com/items?itemName=gregoire.dance

**Comment by varon** [comment]
@xcpky  - See also https://github.com/SuduIDE/ideals
